.eslintrc.js

```diff
@@ -103,6 +103,9 @@ module.exports = {
     'jasmine/no-focused-tests': ['error'],
     'deprecation/deprecation': ['error'],
     '@ngrx/prefer-effect-callback-in-block-statement': 'off',
+    '@typescript-eslint/unbound-method': ['error', {
+      'ignoreStatic': true
+    }],
```

Comment:
It's done for this section https://github.com/France-ioi/AlgoreaFrontend/pull/1697/files#diff-13692c63627680819070ce10027dfa3e558a0817704cb69b2d22a94736873434R59

After research it was declared as best solution